test/CMakeLists.txt

```diff
@@ -304,10 +304,15 @@ foreach(SDK ${SWIFT_SDKS})
           set(test_mode_target_suffix "-${test_mode}")
         endif()
 
+        set(maybe_command_upload_stdlib)
+        if(NOT test_mode STREQUAL "only_non_executable")
+          set(maybe_command_upload_stdlib ${command_upload_stdlib})
+        endif()
+
         set(test_target_name
             "check-swift${test_subset_target_suffix}${test_mode_target_suffix}${VARIANT_SUFFIX}")
         add_custom_target("${test_target_name}"
-            ${command_upload_stdlib}
+            ${maybe_command_upload_stdlib}
```

Comment:
This is a slight change in behaviour, but I don’t think anyone was invoking the tests in “only_non_executable” (at least not from the build script), so hopefully it doesn’t affect any platform. My reasoning is that if the tests are the non executable tests, there’s no need to upload anything.

In Android there doesn’t seem to be a `command_upload_swift_reflection_test`, but I can apply the same solution to that variable.